core/core/scan_test.go

```diff
@@ -1,12 +1,47 @@
 package core
 
 import (
+	"context"
 	"testing"
 
 	"github.com/kubescape/kubescape/v3/core/cautils"
 	"github.com/stretchr/testify/assert"
 )
 
+/*
+// Requires kubernetes cluster running, how to test it?
+func TestGetInterfaces(t *testing.T) {
```

Comment:
why is this one commented out? can you either uncomment or remove it?

Reply:
This test required a Kubernetes cluster running locally. Hence, it was failing on CI. I've removed it. Could you please review again?